numba_rvsdg/tests/simulator.py

```diff
@@ -10,6 +10,9 @@
 )
 
 import builtins
+import sys
+
+PYVERSION = sys.version_info[:2]
```

Comment:
Can this go into `numba_rvsdg/core/utils.py`? Maybe it would be a better place for it?

Reply:
The entire simulator or just the `PYVERSION` declaration ?